docs/server/postgresql_installation.rst

```diff
@@ -0,0 +1,92 @@
+.. _postgresql_installation:
+
+Installation with PostgreSQL
+============================
+
+These instructions provide additional steps for setting up Pootle with
+PostgreSQL.
+
+You should read the :ref:`full installation instructions <installation>` in
+order to install Pootle.
+
+
+.. _postgresql_installation#setting-up-db:
+
+Setting up the database
+-----------------------
+
+Use the :command:`psql` command to create a user and database:
+
+.. code-block:: bash
+
+   $ sudo su postgres  # On Ubuntu, may be different on your system
+   postgres@ $ createuser -P pootle  # This will ask you to define the users password.
+   postgres@ $ createdb --encoding='utf8' --locale=en_US.utf8 --template=template0 --owner=pootle pootledb
+
+
+.. _postgresql_installation#software-requirements:
+
+System software requirements
+----------------------------
+
+In addition to the 
+:ref:`system packages <requirements#packages>` set out in the general
+installation requirements you will also require the PostgreSQL client
+development headers in order to build the Python bindings, eg. on Debian
+Jessie:
+
+.. code-block:: bash
+
+  $ sudo apt-get install postgresql-server-dev-9.4
+
+
+.. _postgresql_installation#install-bindings:
+
+Installing PostgreSQL Python bindings
+-------------------------------------
+
+Once you have
+:ref:`set up and activated your virtual environment <installation#setup-environment>`,
+you will need to install the PostgreSQL bindings.
+
+You can do so as follows:
+
+.. code-block:: bash
+
+  (env) $ pip install psycopg2
+
+
+.. _postgresql_installation#init-config:
+
+Initializing the Configuration
+------------------------------
+
+When
+:ref:`initializing your configuration <installation#initializing-the-configuration>`
+you can specify params to set up your database, eg:
+
+.. code-block:: bash
+
+  (env) $ pip init --db postgresql --db-name pootledb --db-user pootle
```

Comment:
Just curious, shouldn't be using an equals after the flags? Like in `--db=postgresql`?